src/overlays/actors/ovl_En_Nb/z_en_nb.c

```diff
@@ -587,9 +583,9 @@ s32 func_80BC0B98(EnNb* this, PlayState* play, ScheduleOutput* scheduleOutput) {
 
 s32 func_80BC0C0C(EnNb* this, PlayState* play, ScheduleOutput* scheduleOutput) {
     if (!CHECK_EVENTINF(EVENTINF_43)) {
-        SubS_UpdateFlags(&this->stateFlags, EN_NB_FLAG_1 | EN_NB_FLAG_2, EN_NB_FLAG_1 | EN_NB_FLAG_2 | EN_NB_FLAG_4);
+        SubS_UpdateFlags(&this->stateFlags, SUBS_OFFER_MODE_ONSCREEN, SUBS_OFFER_MODE_MAX);
     } else {
-        SubS_UpdateFlags(&this->stateFlags, EN_NB_FLAG_4, EN_NB_FLAG_1 | EN_NB_FLAG_2 | EN_NB_FLAG_4);
+        SubS_UpdateFlags(&this->stateFlags, SUBS_OFFER_MODE_AUTO, SUBS_OFFER_MODE_MAX);
```

Comment:
```suggestion
        SubS_UpdateOfferFlags(&this->stateFlags, SUBS_OFFER_MODE_AUTO, SUBS_OFFER_MODE_MAX);
```
Are these always used with the `SUBS_OFFER_XXX` flags?

Reply:
There is currently 1 use that is possibly not using the `OfferFlags` (it still uses 7 as the unset bits, but the actor doesn't actually use the function, and those bits don't seem to be used anywhere https://github.com/zeldaret/mm/pull/1312#discussion_r1247461152).
The function is general enough it doesn't have to be used with just the `OfferFlags` but that does seem to be how it was used, so I'm actually not really sure what is better.